yt_dlp/extractor/instagram.py

```diff
@@ -293,7 +293,9 @@ def _real_extract(self, url):
         video_id, url = self._match_valid_url(url).group('id', 'url')
         webpage, urlh = self._download_webpage_handle(url, video_id)
         if 'www.instagram.com/accounts/login' in urlh.geturl():
-            self.raise_login_required('You need to log in to access this content')
+            self.report_warning('Main webpage is locked behind the login page.')
```

Comment:
Do we lose metadata with this? If so, explain that in the warning. If not, warning isn't needed

Reply:
Seems like the only fields missing are `uploader` and `comments` (and `duration` but imo it was already kinda broken in the first place)